src/amo/routes.js

```diff
@@ -26,6 +30,12 @@ export default (
     <Route path=":visibleAddonType/:slug/" component={CategoryPage} />
     <Route path="fxa-authenticate" component={HandleLogin} />
     <Route path="search/" component={SearchPage} />
+    <Route path="401/"
+      component={config.get('isDevelopment') ? NotAuthorized : NotFound} />
+    <Route path="404/" component={NotFound} />
```

Comment:
I guess this is technically a bit redundant, but I like telegraphing the intent here to other developers and making it convention that we put our error pages in an easily accessible place for testing.